packages/lune/src/builder.ts

```diff
@@ -52,30 +24,11 @@ export async function buildPlugin(path: string, to: string, cfg: LuneCfg, minify
 
   if (!entryPoint) throw new Error("failed to find entrypoint - check your working directory and config");
 
-  await build({
-    entryPoints: [entryPoint],
-    outfile,
-    bundle: true,
-    minify,
-    plugins: [
-      ...(cfg.prePlugins ?? []),
-      solidPlugin(),
-      (cfg.cssModules
-        ? sassPlugin({
-            style: "compressed",
-            sourceMap: false,
-            transform: postcssModules({
-              localsConvention: "camelCaseOnly",
-              inject: cfg.cssModules === "legacy" ? false : "shelter",
-            } as any),
-          })
-        : sassPlugin({ style: "compressed", sourceMap: false, type: "css-text" })) as any, // bad but version conflicts suck
-      shelterEsbuildResolver(),
-      ...(cfg.postPlugins ?? []),
-    ],
-    globalName: "__lune_temp_global",
-    logLevel: "silent", // we handle errors ourself
-  });
+  if (cfg.builder === "esbuild") {
+    await createEsbuildBuilder(entryPoint, outfile, minify, cfg);
+  } else {
+    await createRolldownBuilder(entryPoint, outfile, minify, root);
```

Comment:
What exactly is root? Should that not be set based on entrypoint? I think it makes far more sense to have both builders take the lune config as is as their input.

Reply:
I just about wanted to do that, `root` is just the `projectRoot` that is passed to lightningcss for css modules. I'll refactor them later.